BraveRewardsUI/Create Wallet/CreateWalletViewController.swift

```diff
@@ -52,25 +56,35 @@ class CreateWalletViewController: UIViewController {
     }
   }
   
+  let observer: LedgerObserver
+  
+  func setupLedgerObserver() {
+    state.ledger.add(observer)
+    observer.walletInitalized = { [weak self] result in
+      guard let self = self else { return }
+      self.createWalletView.createWalletButton.isCreatingWallet = false
+      if result == .ledgerOk || result == .walletCreated {
+        self.show(WalletViewController(state: self.state), sender: self)
+      } else {
+        self.showFailureAlert()
+      }
+    }
+  }
+  
   // MARK: - Actions
   
+  func showFailureAlert() {
+    let alertController = UIAlertController(title: "Error", message: "Oops! Something went wrong. Please try again.", preferredStyle: .alert)
```

Comment:
Localization missing and this alert code is duplicated. Perhaps we should extract it to some common class.
On other hand this code is very small, not a big deal to duplicate it